src/runtime/threading_backend.cc

```diff
@@ -133,22 +133,39 @@ class ThreadGroup::Impl {
           sizeof(cpu_set_t), &cpuset);
 #endif
     }
-    if (exclude_worker0) {  // bind the master thread to core 0
-      cpu_set_t cpuset;
-      CPU_ZERO(&cpuset);
-      if (reverse) {
-        CPU_SET(sorted_order_[sorted_order_.size() - 1], &cpuset);
-      } else {
-        CPU_SET(sorted_order_[0], &cpuset);
+    if (exclude_worker0) {  // master thread run task
+#if defined(__ANDROID__)
+      SetFullCpuAffinity();
+#else
+      const char* bind_master_thread = getenv("TVM_BIND_MASTER_THREAD");
```

Comment:
Will need to document `TVM_BIND_MASTER_THREAD`

Reply:
Will do.